### PR TITLE
MS. Laud Misc. 146: Peter Kidd description

### PR DESCRIPTION
**Pull request by holfordm**

### CODE REVIEW

File: collections/Laud_Misc/MS_Laud_Misc_146.xml [hunk 1]
```diff
@@ -39,42 +42,158 @@
...
+                     <finalRubric>Sic est finis C. presbiter de cele<ex>bratione</ex> missarum summatim repetiti solempniter per Conradum ordinarum <unclear>doc<ex>torem</ex></unclear> Erfordensis. Amen</finalRubric>
+                     <textLang mainLang="la">Latin</textLang>
+                  </msItem>
+                  <msItem n="8" xml:id="MS_Laud_Misc_146-item8">
+                     <locus>(fols. 140v–152r)</locus>
+                     <author key="person_207244871">Henricus de Langenstein</author>
+                     <title key="work_6667">De reductione effectuum spiritualium in virtutes communes</title>
+                     <incipit>Propter Ammirari inceperunt antiquitus homines prophetari</incipit>
+                     <explicit>nos custodiat Marie filius Ihesus Cristus amen</explicit>
+                     <finalRubric>Hec ille - - - Explicit edificium magistri Heinrici de Hassia limpidissimi speculatoris deo laus</finalRubric>
```
Comment (adunning) [inline collections/Laud_Misc/MS_Laud_Misc_146.xml:127]: Is `- - -` an illegible word?

Comment (holfordm) [reply collections/Laud_Misc/MS_Laud_Misc_146.xml:127]: I wondered that as well - it represents a sort of squiggle in the manuscript. Other ideas for representing this?

Comment (adunning) [reply collections/Laud_Misc/MS_Laud_Misc_146.xml:127]: If you need a solid bar, there's the three-em dash: ⸻.

Comment (holfordm) [reply collections/Laud_Misc/MS_Laud_Misc_146.xml:127]: yes, that would probably be clearer

Comment (adunning) [reply collections/Laud_Misc/MS_Laud_Misc_146.xml:127]: I suppose you could also add a `<sic>` tag to clarify that this is not editorial.

**Pull request merged**